common/djangoapps/student/roles.py

```diff
@@ -218,6 +218,12 @@ def users_with_role(self):
         )
         return entries
 
+    def get_org_for_user(self, user):
```

Comment:
```suggestion
    def get_orgs_for_user(self, user):
```
This returns multiple short names, so please make the function name plural.